student.rb

```diff
@@ -1,14 +1,14 @@
 require_relative 'person'
 class Student < Person
-  def initialize(classroom)
+  attr_accessor :classroom
```

Comment:
- [ ] Since we are trying to specify a custom setter method, we just need an `attr_reader` to get the information about the classroom. Please change the accessor to a reader. 🤓️

Reply:
Thanks! the change has been made